app/commands/track/update_build_status.rb

```diff
@@ -0,0 +1,359 @@
+class Track::UpdateBuildStatus
+  include Mandate
+
+  initialize_with :track
+
+  def call
+    track.build_status = build_status
```

Comment:
Should this not be `track.update(...)`. If it's not updating, I'd rather this command was `DetermineTrackStatus` and then the `track.build_status = ` was part of a wider command.

Reply:
No, as `build_status` is not a materialized column of `Track`, but instead automatically assigned to Redis: https://github.com/exercism/website/pull/3017/files#diff-ccd907c771daebc66834912a9e0dca52fe42ac4580297fd82f8e29debdfa6a83

Reply:
Ah - not got there yet :)